src/ActionUnit.h

```diff
@@ -45,7 +45,7 @@ class ActionUnit
     friend class XMLimport;
 
 public:
-                          ActionUnit( Host * pHost ) : mpHost(pHost), mMaxID(0) {;}
+                          ActionUnit( Host * pHost ) : mpHost(pHost), mMaxID(0), mpToolBar(), mpEasyButtonBar(), mModuleMember() {;}
```

Comment:
That is removed in my formatting changes, though it's not something that's automatically removed.